temel_bilgiler/kurulumlar/web_nasıl_çalışır.md

```diff
@@ -0,0 +1,57 @@
+* Sunucu nedir kısaca açıklayınız.
+* IP adreslerinin ne olduğunu açıklayınız.
+* DNS sunucularının ne olduğunu açıklayınız.
+
+### Ödev
+
+<div class="lesson-content__panel" markdown="1">
+  1. İnternetin nasıl çalıştığına dair genel bir bakış için bu [BBC short](https://www.youtube.com/watch?v=eHp1l73ztB8) 'u izleyin.  
+  2. Mozilla'nın "İnternet nasıl çalışır?" başlıklı bu [makalesini](https://developer.mozilla.org/en-US/Learn/Common_questions/How_does_the_Internet_work) okuyun.  
+  3. [How the Internet Works in 5 Minutes](https://youtu.be/7_LPdttKXPc?t=46s) izleyin.  
+  4. Web sayfası, web sunucusu ve arama motoru arasındaki [farkları](https://developer.mozilla.org/en-US/Learn/Common_questions/Pages_sites_servers_and_search_engines) okuyun.  
+  5. Web tarayıcısının ne olduğunu açıklayan bu [Google](https://youtu.be/BrXPcaRlBqo) kısa videosunu izleyin. Ardından, [şu anda](https://www.whatsmybrowser.org/) hangi web tarayıcısını kullandığınızı öğrenin.  
+  6. Web'in bir bölümünün nasıl [diğeriyle etkileşime girdiğini](https://developer.mozilla.org/en-US/Learn/Getting_started_with_the_web/How_the_Web_works#Clients_and_servers) ve bir DNS isteğini nasıl [okuduğunu](https://developer.mozilla.org/en-US/Learn/Common_questions/What_is_a_domain_name#How_does_a_DNS_request_work) veya [izlediğini](https://www.youtube.com/watch?v=72snZctFFtA&feature=youtu.be&t=45s) okuyun.  
+</div>
+
+### Bilgi Ölçme
+
+Bu bölüm, bu dersi anlayıp anlamadığınızı kendi başınıza kontrol etmeniz için sorular içermektedir. Bir soruyu yanıtlamakta sorun yaşıyorsanız, soruya tıklayın ve bağlantının verdiği materyali inceleyin.
+
+*   [What is a web server?](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Pages_sites_servers_and_search_engines)
+*   [What is a network?](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/How_does_the_Internet_work)
+*   [What is the internet?](https://www.youtube.com/watch?v=7_LPdttKXPc&t=46s)
+*   [What is an IP address?](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/How_does_the_Internet_work)
+*   [What is a router?](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/How_does_the_Internet_work)
+*   [What is an ISP?](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/How_does_the_Internet_work)
+*   [What are packets and how are they used to transfer data?](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/How_the_Web_works#packets_explained)
+*   [What is a client?](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/How_the_Web_works#clients_and_servers)
+*   [What is a server?](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/How_the_Web_works#clients_and_servers)
+*   [What is a web page?](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Pages_sites_servers_and_search_engines)
+*   [What is a web browser?](https://www.youtube.com/watch?v=BrXPcaRlBqo&feature=youtu.be)
```

Comment:
fix: * [Web tarayıcısı nedir?](https://www.youtube.com/watch?v=BrXPcaRlBqo&feature=youtu.be)